owncloudTestUtil/src/main/java/com/owncloud/android/testutil/security/PassCodeUtils.kt

```diff
@@ -0,0 +1,24 @@
+/**
+ * ownCloud Android client application
+ *
+ * @author Juan Carlos Garrote Gascón
+ *
+ * Copyright (C) 2021 ownCloud GmbH.
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License version 2,
+ * as published by the Free Software Foundation.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+package com.owncloud.android.testutil.security
+
+const val EXAMPLE_PASSCODE_4_DIGITS = "1111"
+const val EXAMPLE_PASSCODE_6_DIGITS = "111111"
```

Comment:
To keep consistency, I would rename those constants to `OC_PASSCODE_X_DIGITS`

BTW, we use the ownCloudTestUtil module for constants or objects that are used in several modules to not duplicate several times the same constant. At the moment, since we only use the passcode or pattern in the ownCloudApp module, we can include this file directly in the ownCloudApp module.

Reply:
Ok, I changed the name. Related to the module, I've seen there are some other constants that are only used in one module and they are defined also in the ownCloudTestUtil module. I think it's more clear to keep all of them in this utils module, what do you think?

Reply:
Ok, keep them in ownCloudTestUtils